index.html

```diff
@@ -11,11 +11,19 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;700&display=swap" rel="stylesheet"/>
     <script src="https://kit.fontawesome.com/25935046d9.js" crossorigin="anonymous"></script>
   </head>
-
   <body>
    <nav class="nav">
       <p class="logo">chester</p>
-      <img src="images/Union.png" alt="logo" class="hamburger open-menu" />
+      <img src="images/Union.png" alt="logo" class="hamburger open-menu logo-text" />
+      <div class="nav-list">
+        <span class="close-x">&times;</span>
+        <a href="#contact" class="contactlink1">Contact</a> 
+        <div class="borderlink borderlink1"></div>
+        <a href="#about" class="aboutlink1">About</a>
+        <div class="borderlink borderlink2"></div>
+        <a href="#recent-work" class="portfolink1">Portfolio</a>
```

Comment:
- [ ] As required by the previous reviewer, make sure When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed, the `portfolio` nav link still does not woks